api/clients/disperser_client.go

```diff
@@ -20,6 +20,9 @@ import (
 type Config struct {
 	Hostname          string
 	Port              string
+	// BlobDispersal Timeouts for both authenticated and unauthenticated dispersals
```

Comment:
v2 might have an architecture change on how authentication will be done. We can revisit at that time. For now, the latency is dominated by data size, so timeout should be identical I think

Reply:
sgtm. will resolve this as there's already a ton of comments on this PR. But let me know if you wanted me to add a TODO comment in there or something.